app.js

```diff
@@ -139,6 +139,7 @@ const assistant = new Assistant({
       return;
     }
     const { channel, thread_ts } = message;
+    const { userId, teamId } = context;
```

Comment:
note: Optional, but I'm getting the `userId` and `teamId` from the context to provide it as the `recipient_user_id` and `recipient_team_id` in the text stream. This isn't required in a DM experience, but it's good practice.